### PR TITLE
v2podman ps revert structure changes

### PR DESCRIPTION
**Pull request by baude**
reverting name changes to the listcontainer structure because it negatively impacted the direct consumption of the restful API.  instead we now use a local structure in the CLI to modify the output as needed.

Signed-off-by: Brent Baude <bbaude@redhat.com>

### CODE REVIEW

**Comment by mheon** [comment]
LGTM

File: pkg/ps/ps.go [hunk 1]
```diff
@@ -158,11 +158,11 @@ func ListContainerBatch(rt *libpod.Runtime, ctr *libpod.Container, opts entities
...
-		ContainerNames: []string{conConfig.Name},
+		Names:          []string{conConfig.Name},
 		Pid:            pid,
 		Pod:            conConfig.Pod,
-		PortMappings:   conConfig.PortMappings,
-		ContainerSize:  size,
+		Ports:          conConfig.PortMappings,
+		Size:           size,
 		StartedAt:      startedTime.Unix(),
 		ContainerState: conState.String(),
```
Comment (marusak) [inline pkg/ps/ps.go:167]: Also `ContainerState` was `State` before.

File: test/apiv2/20-containers.at [hunk 2]
```diff
@@ -21,8 +21,8 @@ t GET libpod/containers/json?all=true 200 \
   length=1 \
   .[0].Id~[0-9a-f]\\{12\\} \
   .[0].Image=$IMAGE \
-  .[0].Cmd[0]="true" \
-  .[0].ContainerState~\\\(exited\\\|stopped\\\) \
+  .[0].Command[0]="true" \
+  .[0].State~\\\(exited\\\|stopped\\\) \
```
Comment (marusak) [inline test/apiv2/20-containers.at:25]: This should catch it.

**Comment by marusak** [comment]
Thanks, now it is usable as it was before. LGTM

**Pull request merged**